api/reference/conditions.adoc

```diff
@@ -0,0 +1,156 @@
+
+[[conditions]]
+== Conditions
+
+Management of Conditions is provided via the following collection:
+
+[source,data]
+----
+/api/conditions
+----
+
+Full CRUD actions on conditions is available:
+
+* link:#querying-conditions[Querying Conditions]
+* link:#creating-conditions[Creating Conditions]
+* link:#editing-conditions[Editing Conditions]
+* link:#deleting-conditions[Deleting Conditions]
+
+[[querying-conditions]]
+=== Querying Conditions
+
+Querying all conditions in the system is simply:
+
+----
+GET /api/conditions
+----
+
+Getting details on a specific Condition:
+
+----
+GET /api/conditions/:id
+----
+
+[[creating-conditions]]
+=== Creating Conditions
+
+Conditions can be created via a POST to the conditions collection or via the *create* action
+signature which also allows creation of multiple conditions in a single request.
+
+----
+POST /api/conditions
+----
+
+[source,json]
+----
+{
+  "name" : "sample_condition",
+  "description" : "Sample Condition",
+  "expression" : { "=" : { "field" : "ContainerImage-architecture", "value" : "dsa" } },
```

Comment:
Yeah, it's a valid field, I'm just wondering if the expression is valid overall. If the target collection is ExtManagementSystem I'd expect it to be expressed in terms of that